src/libYARP_pcl/include/yarp/pcl/Pcl.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2006-2018 Istituto Italiano di Tecnologia (IIT)
+ * Copyright (C) 2006-2010 RobotCub Consortium
+ * All rights reserved.
+ *
+ * This software may be modified and distributed under the terms of the
+ * BSD-3-Clause license. See the accompanying LICENSE file for details.
+ */
+
+#ifndef YARP_PCL_PCL_H
+#define YARP_PCL_PCL_H
+
+#include <pcl/io/pcd_io.h>
+#include <yarp/sig/PointCloud.h>
+
+namespace yarp
+{
+namespace pcl
+{
+
+/**
+ * Convert a yarp::sig::PointCloud to a pcl::PointCloud object
+ * @param yarpCloud yarp::sig::PointCloud input
+ * @return a pcl PointCloud filled with data contained in the yarp cloud.
+ */
+template< class T1, class T2 >
+inline bool toPCL(yarp::sig::PointCloud< T1 > &yarpCloud, ::pcl::PointCloud< T2 > &pclCloud)
```

Comment:
If `yarpCloud` is just an input argument, perhaps we can have `const yarp::sig::PointCloud< T1 > &yarpCloud`?